packages/zen-bull-vault/src/LeverageZen.sol

```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: GPL-3.0-only
+//SPDX-License-Identifier: BUSL-1.1
```

Comment:
``StrategyMath.sol`` is imported here which is ``//SPDX-License-Identifier: AGPL-3.0-only
``, doesn't this mean we have to make it GPL?

Reply:
It also imports our ``UniOracle.sol``, which I think needs to be licensed as GPL, but maybe thats ok?